controllers/booking.js

```diff
@@ -80,7 +76,7 @@ async function createBooking(req, res) {
 }
 
 // @desc    Update a booking by ID
-// @route   PUT /booking/:id/edit
+// @route   POST /booking/:id
```

Comment:
current update operation works but isn't ideal as it doesn't enable changes to just one field, for eg, no of Pax. think this is so because the other fields (eg, dateTime) are specified as compulsory in daos. understand we need those fields to be compulsory for createBooking... one way to resolve this could be by making the field compulsory front end (instead of backend). Pls review this!

Reply:
Doesn't the request work as what you call ideal? Values of other fields remain.
<img width="858" alt="Screenshot 2024-01-15 at 9 36 24 AM" src="https://github.com/jx0906/proj3-backend/assets/88537845/0df5ec2a-1033-45ba-88a7-d6f528ad8898">

Reply:
i didn't manage to get this earlier. had assumed some errors with my submission since that was the Postman response I got everytime I had an errorneous req.body. thank you for the screenshot. I retried and it worked now. suspect my earlier req.body was still wrong (had an extra comma).. sorry to hold you up!